distutils/ccompiler.py

```diff
@@ -989,7 +989,8 @@ def _make_relative(base):
         # Chop off the drive
         no_drive = os.path.splitdrive(base)[1]
         # If abs, chop off leading /
-        return no_drive[os.path.isabs(no_drive) :]
+        is_abs = os.path.isabs(no_drive) or sys.platform == 'win32' and (no_drive.startswith('/') or no_drive.startswith('\\'))
```

Comment:
This can be simplified slightly:
```suggestion
        is_abs = os.path.isabs(no_drive) or sys.platform == 'win32' and no_drive.startswith(('/', "\\"))
```